### PR TITLE
fix: Add ready function type definition

### PR DESCRIPTION
**Pull request by huajun07**
Added `ready` function into type definition.

Fixing #294

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -47,4 +48,4 @@ export interface Options {
     idleTimeoutMillis?: number;
 }
 
-export function createPool<T>(factory: Factory<T>, opts?: Options): Pool<T>;
+export function createPool<T>(factory: Factory<T>, opts?: Options): Pool<T>;
```
Comment (sandfox) [inline index.d.ts:51]: minor nitpick, can you please re-add the newline at the end.

**Comment by Kikobeats** [comment]
Thanks!

**Comment by nwalters512** [comment]
I just came to GitHub looking for this! @Kikobeats do you know when this change will be released? It looks like the last version was released over one year ago.

**Comment by Kikobeats** [comment]
done https://github.com/coopernurse/node-pool/releases/tag/v3.9.0

**Comment by nwalters512** [comment]
Thanks!

**Pull request merged**